src/test/java/org/gridsuite/modification/server/utils/MatcherTwoWindingsTransformerCreationInfos.java

```diff
@@ -45,8 +57,19 @@ public boolean matchesSafely(TwoWindingsTransformerCreationInfos m) {
                 && m.getMagnetizingConductance() == reference.getMagnetizingConductance()
                 && m.getSeriesReactance() == reference.getSeriesReactance()
                 && m.getSeriesResistance() == reference.getSeriesResistance()
+                && areDoublesEqual(m.getRatedS(), reference.getRatedS())
                 && matchesCurrentLimitsInfos(m.getCurrentLimits1(), reference.getCurrentLimits1())
-                && matchesCurrentLimitsInfos(m.getCurrentLimits2(), reference.getCurrentLimits2());
+                && matchesCurrentLimitsInfos(m.getCurrentLimits2(), reference.getCurrentLimits2())
+                && matchesRatioTapChangerInfos(m.getRatioTapChanger(), reference.getRatioTapChanger())
+                && matchesPhaseTapChangerInfos(m.getPhaseTapChanger(), reference.getPhaseTapChanger());
+    }
+
+    public boolean areDoublesEqual(Double a, Double b) {
```

Comment:
hum... strange to have this

Reply:
Agreed, but since Double can be null and == operator does not compare the value itself I have to do extra checks before calling equals method